clustered/client/src/main/java/org/ehcache/clustered/client/config/builders/ClusteringServiceConfigurationBuilder.java

```diff
@@ -90,18 +114,19 @@ public ServerSideConfigurationBuilder expecting() {
    *
    * @throws NullPointerException if {@code unit} is {@code null}
    * @throws IllegalArgumentException if {@code amount} is negative
+   *
+   * @deprecated Use {@link #timeouts(Timeouts)}. Note that calling this method will override any timeouts previously set
+   * by setting the read operation timeout to the specified value and everything else to its default.
    */
+  @Deprecated
   public ClusteringServiceConfigurationBuilder readOperationTimeout(long duration, TimeUnit unit) {
-    return new ClusteringServiceConfigurationBuilder(this, TimeoutDuration.of(duration, unit));
+    Duration readTimeout = Duration.of(duration, toChronoUnit(unit));
+    return timeouts(Timeouts.builder().setReadOperationTimeout(readTimeout).build());
```

Comment:
Why can't we add a copying builder and just override the read operation timeout?

Reply:
Because it's deprecated. So anybody currently using this will have the same behaviour, and if he wants to add move timeouts, migrating would be a good idea.